cmds/contest-generator/README.md

```diff
@@ -0,0 +1,63 @@
+## ConTest Generator
+
+`contest-generator` is a tool used to build ConTest with the plugins you want.
+
+We provide a standard [main.go](/cmds/contest/main.go), where we only include
+the most common plugins.
+
+If you need to build `ConTest` with a custom set of plugins without maintaining
+your own `main.go` program, you can `contest-generator`.
```

Comment:
I think there should be a verb somewhere here
```suggestion
your own `main.go` program, you use can `contest-generator`.
```

Reply:
done